htdocs/install/mysql/migration/18.0.0-19.0.0.sql

```diff
@@ -201,3 +201,5 @@ ALTER TABLE llx_commande_fournisseur_dispatch ADD COLUMN element_type varchar(50
 ALTER TABLE llx_expensereport DROP INDEX idx_expensereport_fk_refuse, ADD INDEX idx_expensereport_fk_refuse(fk_user_refuse);
 
 INSERT INTO llx_c_forme_juridique (fk_pays, code, libelle) VALUES (1,'66','Société publique locale');
+
+ALTER TABLE llx_website ADD COLUMN name_template varchar(255) NULL;
```

Comment:
Because the PR will be merge in develop for 20.0, can you put the new line into file 
19.0.0-18.0.0.sql
Also you must update in samePR the file llx_website.sql  (to have field created also for first install)

Reply:
should i create the sql file 19_18 because i didn't found it